Sources/BugsnagPerformance/Private/BugsnagPerformanceCrossTalkAPI.mm

```diff
@@ -11,6 +11,8 @@
 
 @implementation BugsnagPerformanceCrossTalkAPI
 
+static BugsnagPerformanceConfiguration *configuration;
```

Comment:
I'm of two minds about this...

On the one hand, it's a return value from a global API so it feels more global-y...

On the other hand, it's an injected value, and so technically should be a member of `BugsnagPerformanceCrossTalkAPI`...

I'm leaning more towards "member", but it could be argued either way, I suppose.